### ISSUE TITLE
ALC889 possible with CSR activated under 10.11?

### ISSUE THREAD

**Issue by mperkh** [toleda/audio_RealtekALC#25]
Hi,
first thanks for all the great tools and instructions found here.

My system:
Z68X-UD3H-B3 (ALC889 onboard audio) with i7-2600k cpu (HD3000) and using latest Clover.

Question:
I want to run El Capitan with CSR active (=not having the in the Readme.txt mentioned requirement of `rootless=0`).

Is it possible make Realtek ALC889 work, by just modifying Clover's `config.plist`, patch the DSDT and putting Kexts in `/EFI/CLOVER/kexts/10.11`? So no unsigned kexts are in S/L/E or /L/E?

**Comment by toleda** [comment]
No.  The bootloader does not configure the codec, OS X does. Codec specific layout and pathmap are loaded with AppleHDA binary. Once installed, set csr=0.  csr=3 is only required to install system files and rebuild kernel cache for unsigned kexts, OS X installs and updates.
